docs/controls/ImageEx.md

```diff
@@ -43,9 +43,9 @@ The [ImageEx](/dotnet/api/microsoft.toolkit.uwp.ui.controls.imageex) control dow
 
 ImageEx control supports use of Progress Indicator. This can be enabled by adding ImageEx template from previous release of the control.
 
-- [ImageEx Control with Progress Indicator XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/rel/7.0.0/Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.xaml) is the XAML template used in v.7.0.0 of toolkit.
+- [ImageEx Control with Progress Indicator XAML File](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/rel/2.2.0/Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageEx.xaml) is the XAML template used in v.2.2.0 of toolkit.
```

Comment:
```suggestion
```

Not sure why we removed the progress ring in the past, but if we don't support the feature anymore, we should just remove this from the docs entirely.,